.github/workflows/forest.yml

```diff
@@ -402,6 +402,15 @@ jobs:
       # We use a custom Dockerfile for CI to speed up the build process.
       FOREST_DOCKERFILE_OVERRIDE: scripts/devnet/forest_ci.dockerfile
     steps:
+      - name: Maximize build space
+        uses: easimon/maximize-build-space@master
```

Comment:
Do we have to use @master here? Does `v10` not work?

Reply:
None of them work. :)